bin/ncp-diag

```diff
@@ -66,78 +66,52 @@ echo "Internet check|$( ping -W 2 -w 1 -q github.com &>/dev/null && echo ok || e
 
 function is_port_open()
 {
-  local port_url tmp_file token ipv4_portcheck_args ipv6_portcheck_args
-  tmp_file=$(mktemp)
-  trap 'rm -rf ${tmp_file}' EXIT ERR SIGINT SIGQUIT SIGABRT SIGTERM SIGHUP
-
-  local port="${1?}"
-  local publicIPv4="${2}"
-  local publicIPv6="${3}"
-
-  readonly port_url="https://portchecker.co"
-
-  if [[ -z "$publicIPv4" ]] && [[ -z "$publicIPv6" ]]
-  then
-    echo -n "Error - IPv4 & IPv6: [N/A] Couldn't get public IP."
-    return 1
-  fi
-
-  token=$(wget -T2 -t1 -qO- --keep-session-cookies --save-cookies "${tmp_file}" "${port_url}" | grep -oP "_csrf\" value=\"\K.*\"" )
-  readonly ipv4_portcheck_args=(-T2 -t1 -qO- --load-cookies "${tmp_file}" "${port_url}/check" --post-data "target_ip=${publicIPv4}&port=${port}&_csrf=${token::-1}")
-  readonly ipv6_portcheck_args=(-T2 -t1 -qO- --load-cookies "${tmp_file}" "${port_url}/check" --post-data "target_ip=${publicIPv6}&port=${port}&_csrf=${token::-1}")
-
-  [[ -n "${token}" ]] || {
-    echo -n "Error - Couldn't obtain a token for port check"
-    return 1
-  }
-
-  local ipv4_port_access=False
-  local ipv6_port_access=False
-  [[ -n "$publicIPv4" ]] && \
-    grep -q '<span class="green">open</span>' <(wget "${ipv4_portcheck_args[@]}") && \
-    ipv4_port_access=True
-  [[ -n "$publicIPv6" ]] && \
-    grep -q '<span class="green">open</span>' <(wget "${ipv6_portcheck_args[@]}") && \
-    ipv6_port_access=True
-
-  local result=""
-  if [[ "${ipv4_port_access}" == True ]] || [[ "${ipv6_port_access}" == True ]]
-  then
-    result="open ("
-  else
-    result="closed"
-  fi
-
-  [[ "${ipv4_port_access}" == True ]] && result="${result}ipv4)"
-
-  [[ "${ipv6_port_access}" == True ]] && result="${result/)/ \& }ipv6)"
+  # The URL leads to an application I've deployed for NCP on https://fly.io using a Docker container I made.
+  # The image for the container is available on Docker Hub (zendai/checkport:sanic) if you wish to deploy one yourself.
+  # The code for the Sanic server and Docker image is available at: https://github.com/ZendaiOwl/Build/tree/master/Docker/Python/Sanic/checkport
+  # I only have a free tier with limited outbound data per month, 100GB p/month.
+  # If we go over 100GB outbound data in a month, I will start being charged for the data going over that limit.
+  # I used a low level Python socket library & fortunately each request only consumes aprox. ~ 60-74 bytes p/second.
+  # Meaning 100GB should be plenty, it should be enough to handle a little less
+  # than 450 request p/second a month, unless my calculations are wrong.
+  # Thank you :pray: from Victor-ray, S. https://github.com/ZendaiOwl
+  local -r PORTURL="https://checkport.zendai.net.eu.org/check"
+  local TYPE="${1?}" RESULT=("")
+  # Checks both port 80 & 443 for IPv4/IPv6 and returns the result or [N/A] [N/A]
+  test "$TYPE" -eq 4 && RESULT=("$(curl --silent --max-time 4 --ipv4 "$PORTURL" | jq -r '."80",."443"' )")
```

Comment:
I would prefer to use the bash bracket syntax over the literal test command for consistency with our code base (e.g. `[[ "$TYPE" == 4 ]]` or `[[ "$TYPE" -eq 4 ]]`)

Reply:
Sure, I can change that 👍